src/test/ui/issues/issue-69396-const-no-type-in-macro.stderr

```diff
@@ -17,10 +17,10 @@ LL | | }
    = note: this error originates in the macro `suite` (in Nightly builds, run with -Z macro-backtrace for more info)
 
 error: missing type for `const` item
-  --> $DIR/issue-69396-const-no-type-in-macro.rs:4:19
+  --> $DIR/issue-69396-const-no-type-in-macro.rs:4:20
    |
 LL |               const A = "A".$fn();
-   |                     ^ help: provide a type for the constant: `A: usize`
+   |                      ^ help: provide a type for the constant: `: usize`
```

Comment:
Looks kinda off, IMHO: suggestion talks about type but suggest not var+type, not type, but something in between colon+type.

Reply:
I think old style is better, and also seems more consistant with other suggestions in rustc.

Reply:
If anything this could be fixed by bumping to a verbose diagnostic, but I don't think this is necessarily a regression.